build/testdata/060.golden

```diff
@@ -28,4 +28,14 @@ strings = [
     "'\"foo\"'",
     '''\'\'\'"""foo"""\'\'\'''',
     """'''\"\"\"foo\"\"\"'''""",
+
+    # raw strings
+    r'\foo',
+    r'"\foo',
+    r"\foo",
+    r"'\foo",
+    r'''\foo''',
```

Comment:
why not double-quotes here?

Reply:
Replaced with double quotes (but only if there are no single quotes inside at all). E.g. `r''' ' '''` won't be reformatted as `r""" ' """` (even if in this case it's possible).